pages/_document.js

```diff
@@ -1,24 +1,31 @@
-// pages/_document.js
-
 import Document, { Html, Head, Main, NextScript } from "next/document";
 
 export default function StreamlitDocument() {
   const setInitialTheme = `
```

Comment:
Add a todo to use `useEffect` here when we refactor this next time.

Reply:
Added on 200e1a3